src/components/Accordion/Accordion/__stories__/accordion.stories.mdx

```diff
@@ -0,0 +1,149 @@
+  "Default state of accordion is closed unless used for navigation",
+  "Accordion content can include icons, radio buttons, and checkboxes.",
+]}/>
+
+## Variants
+### Multi active
+Each section can be expanded without closing the others
+<Canvas>
+  <Story name="Multi active">
+    <div className="monday-storybook-accordion_small-wrapepr">
+      <Accordion allowMultiple defaultIndex={[1, 3]}>
+        <AccordionItem title="Notifications"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Setting"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Connectivity"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Integration"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Assets"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+      </Accordion>
+    </div>
+  </Story>
+</Canvas>
+
+### Single active
+Only one section can be open at the time
+<Canvas>
+  <Story name="Single active">
+    <div className="monday-storybook-accordion_small-wrapepr">
+      <Accordion defaultIndex={[1]}>
+        <AccordionItem title="Notifications"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Setting"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Connectivity"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Integration"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Assets"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+      </Accordion>
+    </div>
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      className: "monday-storybook-accordion_big-figure",
+      positive: {
+        component:
+          <Accordion className="monday-storybook-accordion_box-small">
+            <AccordionItem title="Notifications" />
+            <AccordionItem title="Security" />
+            <AccordionItem title="InfoAssets" />
+          </Accordion>,
+        description:"Use informative short labels."
+      },
+      negative: {
+        className: "monday-storybook-accordion_big-figure",
+        component:
+          <Accordion className="monday-storybook-accordion_box-small">
+            <AccordionItem title="Closed" />
+            <AccordionItem title="Closed" />
+            <AccordionItem title="Closed" />
+          </Accordion>,
+        description:"Indicate the accordion state with labels."
+      }
+    }
+  ]}
+/>
+
+## Use cases and examples
+### Preferences Accordion
+<Canvas>
+  <Story name="Preferences Accordion">
+    <div className="monday-storybook-accordion_wrapper">
```

Comment:
please set the accordion width by using the className prop instead of by using a wrapper